Android build failure with v2.2.0 - RNDatePickerDialogFragment.java:105: error: missing return statement

# Bug

Android fails to build version 2.2.0.

`> Task :@react-native-community_datetimepicker:compileReleaseJavaWithJavac FAILED
/data/workspace/AndroidReactBuildNew/node_modules/@react-native-community/datetimepicker/android/src/main/java/com/reactcommunity/rndatetimepicker/RNDatePickerDialogFragment.java:105: error: missing return statement
  }
  ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
1 error
`

## Environment info

React native info output:

```bash
  IDEs:
    Android Studio: 3.4 AI-183.6156.11.34.5522156
    Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
  npmPackages:
    @react-native-community/cli: ^1.9.5 => 1.12.0 
    react: 16.8.6 => 16.8.6 
    react-native: 0.60.4 => 0.60.4

```

Library version: 2.2.0

## Steps To Reproduce

Try to build.

Describe what you expected to happen:

App to build successfully.

## Reproducible sample code

--- a/android/src/main/java/com/reactcommunity/rndatetimepicker/RNDatePickerDialogFragment.java
+++ b/android/src/main/java/com/reactcommunity/rndatetimepicker/RNDatePickerDialogFragment.java
@@ -17,6 +17,8 @@ import android.content.DialogInterface.OnDismissListener;
 import android.content.DialogInterface.OnClickListener;
 import android.os.Build;
 import android.os.Bundle;
+
+import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.DialogFragment;
 import android.widget.DatePicker;
@@ -47,7 +49,8 @@ public class RNDatePickerDialogFragment extends DialogFragment {
     instance.updateDate(date.year(), date.month(), date.day());
   }
 
-  static DatePickerDialog getDialog(
+  static @NonNull
+  DatePickerDialog getDialog(
           Bundle args,
           Context activityContext,
           @Nullable OnDateSetListener onDateSetListener) {
@@ -80,7 +83,7 @@ public class RNDatePickerDialogFragment extends DialogFragment {
                   year,
                   month,
                   day);
-        case DEFAULT:
+        default:
           return new RNDismissableDatePickerDialog(
                   activityContext,
                   onDateSetListener,

